llvm/docs/JITLink.rst

```diff
@@ -11,7 +11,7 @@ Introduction
 This document aims to provide a high-level overview of the design and API
 of the JITLink library. It assumes some familiarity with linking and
 relocatable object files, but should not require deep expertise. If you know
-what a section, symbol, and relocation are you should find this document
+what a section, symbol, and relocation then you should find this document
```

Comment:
substituting "then" for "are" here doesn't work, but it would make sense to do both:

"if you know what a section, symbol, and relocation are then you should find this document..."